composer.json

```diff
@@ -35,31 +36,49 @@
                     "reference": "master"
                 }
             }
+        },
+        {
+            "type": "package",
+            "package": {
+                "name": "noli42/chosen",
+                "version": "3.0.0",
+                "type": "drupal-library",
+                "extra": {
+                    "installer-name": "chosen"
+                },
+                "dist": {
+                    "url": "https://github.com/noli42/chosen/releases/download/3.0.0/chosen-assets-v3.0.0.zip",
+                    "type": "zip"
+                }
+            }
         }
     ],
     "require": {
         "php": ">=8.2",
-        "acquia/blt": "^13.5",
-        "composer/installers": "^1.9",
+        "acquia/blt": "^13.7",
+        "composer/installers": "^1.9 || ^2.0",
         "cweagans/composer-patches": "^1.6",
-        "drupal/acquia_connector": "dev-4.x",
+        "drupal/acquia_connector": "^4.0",
         "drupal/acquia_purge": "^1.1",
-        "drupal/core": "^10",
+        "drupal/allowed_formats": "^3.0",
```

Comment:
This is in `cardinal_service_profile` -- why add it here?

Reply:
@joegl I'll create a different PR with this removed.

Reply:
@joegl Never mind. I got rid of it.